### PR TITLE
api fix and faqs additions

### PR DESCRIPTION
**Pull request by brettguenther**
## What?

Additions to faqs and api docs.

## Why?

One correction and the other changes are gaps.

### CODE REVIEW

File: docs/faqs.mdx [hunk 1]
```diff
@@ -49,6 +49,12 @@ You can use [Regions](/regions) to detect a user's country or state based on lat
 
 Alternatively, you can use the [IP geocoding API](/api#ip-geocode) to detect a user's country or state based on IP address. You can check the `proxy` flag in the response to determine if the user is using a known proxy, VPN, data center, or Tor exit node.
 
+### Is Radar still able to track location in the background if the app is killed?
+
+Yes, if background location permissions have been granted. If the app is killed on iOS (i.e. user swipes app away from the tray), the SDK is able to wake back up and start collecting location again if it receives certain types of location updates from the OS. These services can be activated on iOS by ensuring you have enabled the [location updates background mode](/sdk/ios#background-modes) and are tracking using at least one of following [tracking options](/sdk/tracking): `useStoppedGeofence`, `syncGeofences`, `useMovingGeofence`, `useSignificantLocationChanges` and `useVisits`.
+
+On android, Radar will continue to receive location updates if the app is swiped away, but not if the app is force killed (either manually by the user or via the phone manufacturer aggressively killing background services).
```
Comment (jrtibbetts) [inline docs/faqs.mdx:56]: **A**ndroid

File: docs/faqs.mdx [hunk 2]
```diff
@@ -61,7 +67,11 @@ The Radar [iOS](/sdk/ios) and [Android](/sdk/android) SDKs automatically generat
...
+On the Users detail page, verify the time that the last location update was sent to Radar via the Location section. View recent location updates and events on the [User Activity](/dashboard#user-activity) page. If location updates are not delivering at the expected interval, verify your [tracking options](/sdk/tracking) are correct and being applied properly. If events are not triggering, verify geofence [entry and exit settings](/geofences#stop-detection). **Note that a device needs to exit a geofence before generating a subsequent entry event. So, ensure that the device has been detected outside of the geofence prior to testing another entry.**
```
Comment (jrtibbetts) [inline docs/faqs.mdx:70]: > subsequent entry event. So, ensure that

subsequent entry event, so ensure that

(I know that this wording was already there before your edits.)

**Review by jrtibbetts — APPROVED** [review]

**Comment by brettguenther** [comment]
https://github.com/radarlabs/documentation/issues/183

File: docs/faqs.mdx [hunk 3]
```diff
@@ -49,6 +49,12 @@ You can use [Regions](/regions) to detect a user's country or state based on lat
 
 Alternatively, you can use the [IP geocoding API](/api#ip-geocode) to detect a user's country or state based on IP address. You can check the `proxy` flag in the response to determine if the user is using a known proxy, VPN, data center, or Tor exit node.
 
+### Is Radar still able to track location in the background if the app is killed?
+
+Yes, if background location permissions have been granted. If the app is killed on iOS (i.e. user swipes app away from the tray), the SDK is able to wake back up and start collecting location again if it receives certain types of location updates from the OS. These services can be activated on iOS by ensuring you have enabled the [location updates background mode](/sdk/ios#background-modes) and are tracking using at least one of following [tracking options](/sdk/tracking): `useStoppedGeofence`, `syncGeofences`, `useMovingGeofence`, `useSignificantLocationChanges` and `useVisits`.
```
Comment (jsani-radar) [inline docs/faqs.mdx:54]: might be exposing my ignorance here - isn't swiping an app up away from the tray the same as force killing on iOS? if so, what's the distinction between swiping away / force killing manually? both are mentioned below for Android

File: docs/faqs.mdx [hunk 4]
```diff
@@ -61,7 +67,11 @@ The Radar [iOS](/sdk/ios) and [Android](/sdk/android) SDKs automatically generat
...
+On the Users detail page, verify the time that the last location update was sent to Radar via the Location section. View recent location updates and events on the [User Activity](/dashboard#user-activity) page. If location updates are not delivering at the expected interval, verify your [tracking options](/sdk/tracking) are correct and being applied properly. If events are not triggering, verify geofence [entry and exit settings](/geofences#stop-detection). **Note that a device needs to exit a geofence before generating a subsequent entry event. So, ensure that the device has been detected outside of the geofence prior to testing another entry.**
+
+### Why am I not seeing rate limit errors when tracking location?
```
Comment (jsani-radar) [inline docs/faqs.mdx:72]: should this be "why am I seeing" instead of the negative framing?

**Review by jsani-radar — COMMENTED** [review]
small comment - looking good overall

will merge this by EoW when I can take a closer look at FAQ copy

File: docs/faqs.mdx (hunk 3 above)
Comment (brettguenther) [reply docs/faqs.mdx:54]: swiping away kills the activity on android, but not services. this is just a different architecture + lifecycle on android than iOS

File: docs/faqs.mdx (hunk 4 above)
Comment (brettguenther) [reply docs/faqs.mdx:72]: good catch

**Review by jsani-radar — APPROVED** [review]

**Pull request merged**